db/models/products.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Product = db.define('products', {
+  name : {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  description: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
+  categories: {
+    //tbd - do we want to keep it an array or make it into a concat string and then use a getter method
```

Comment:
would be simplest for admin to be able to enter categories as a comma-separated string & then have a custom method to convert to array